src/test/java/com/faforever/client/util/TimeServiceTest.java

```diff
@@ -0,0 +1,89 @@
+package com.faforever.client.util;
+
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.preferences.ChatPrefs;
+import com.faforever.client.preferences.DateInfo;
+import com.faforever.client.preferences.LocalizationPrefs;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.FormatStyle;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@Execution(ExecutionMode.CONCURRENT)
+@ExtendWith({MockitoExtension.class})
+class TimeServiceTest {
```

Comment:
We should test with other locales as well

Reply:
So which combinations you would like to see? 2(locales) x 3(format styles Auto,Day,Month) ? Or you want to add additional FormatStyles as well?

Reply:
I would test with the us which is m/d/y and another locale which is default d/m/y for each of the three date info options